Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientlocationquery/entity/proxy/descriptor/EntityPatientLocationQuerySecuredServicePortDescriptor.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.fha.nhinc.patientlocationquery.entity.proxy.descriptor;
+
+import gov.hhs.fha.nhinc.entitypatientlocationquerysecured.EntityPatientLocationQuerySecuredPortType;
+import gov.hhs.fha.nhinc.messaging.service.port.ServicePortDescriptor;
+import javax.xml.ws.soap.SOAPBinding;
+
+public class EntityPatientLocationQuerySecuredServicePortDescriptor
+    implements ServicePortDescriptor<EntityPatientLocationQuerySecuredPortType> {
```

Comment:
Should extends SOAP12ServicePortDescriptor instead of ServicePortDescriptor so that you don't need to override getSOAPBindingVersion